### DISCUSSION TITLE
[low priority] Show options while pausing in lazer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30086] [Ideas] [answered]
While pausing in stable it is possible to open and change the beatmap settings.
![screenshot174](https://github.com/user-attachments/assets/09b163a3-542f-4b57-9800-c84f8c92f30d)
Would be nice to have in lazer as well.
Put this as a low priority idea since it would be nice to have but shouldn't be too much of an inconvenience to be missing.

**Comment by gizmogoat** [comment]
I feel like this probably won't be added, since you can change every setting in the settings flyout by pressing Ctrl-O anyway

**Reply by ghost** [reply]
Sure, however people have requested to change some of the settings to be only for that map. So for example, if you cange the dim on a single map right now it changed the dim for all maps, however players have asked to handle this like stable where you can have a default dim setting for all maps but if you change the dim in a single difficulty that this change will only be for that difficulty. Other options could follow suit in the future as well. So that is why I still think of this as a good idea. Again, this should be a low priority thing, but is definitely a good idea imo

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/3853
